app/main/views/services.py

```diff
@@ -633,6 +640,25 @@ def remove_subsection(framework_slug, lot_slug, service_id, section_id, question
     question_to_remove = content.get_question_by_slug(question_slug)
     fields_to_remove = question_to_remove.form_fields
 
+    # simply defining this as an inner function so we can avoid having to manually pass around the umpteen variables
+    # it depends on instead pulling them from the outer scope
+    def set_remove_last_subsection_error_message():
+        flash(REMOVE_LAST_SUBSECTION_ERROR_MESSAGE.format(
+            section_name=containing_section.name.lower(),
+            service_name=(draft.get("serviceName") or draft.get("lotName")).lower(),
+        ), "error")
+        flash("/".join((
```

Comment:
Not sure if I like `"/".join()` better than the previous `"{}/{}".format()`, but only a minor quibble.